cmd/kubeadm/app/cmd/reset.go

```diff
@@ -69,7 +75,7 @@ func NewCmdReset(in io.Reader, out io.Writer) *cobra.Command {
 	)
 
 	cmd.PersistentFlags().StringVar(
-		&criSocketPath, "cri-socket", kubeadmapiv1alpha3.DefaultCRISocket,
+		&criSocketPath, "cri-socket", "",
```

Comment:
Might be adding a something to the flag description like e.g. this parameter should be used only in case automatic cri detection fails